gatling-load-test/README.md

```diff
@@ -0,0 +1,15 @@
+## Parameters
```

Comment:
## Load test

### Scenario
Send payments from customers to merchants ...

### Server configuration
...
### Load test results

### Conclusion
Explain here about your bottleneck during concurrency